nucliadb_models/nucliadb_models/file.py

```diff
@@ -60,6 +65,11 @@ class FileField(BaseModel):
     file: FileB64
 
 
+class ExternalFileField(BaseModel):
+    uri: str
+    extra_headers: Dict[str, str] = {}
```

Comment:
I think this field should be `Optional`, public links may not need extra headers

Reply:
yup! The fact that it has a default value makes it "optional" in terms of API: if the user doesn't set it, it won't break.
However, adding the `Optional` pydantic type means that it can be `None` too, which I don't want to.

Reply:
Yes, my mistake